source/_integrations/panasonic_viera.markdown

```diff
@@ -11,79 +11,60 @@ ha_domain: panasonic_viera
 
 The `panasonic_viera` platform allows you to control a Panasonic Viera TV.
 
-Currently known supported models:
+## Configuration
+To configure your Panasonic Viera TV, head to the **Configuration > Integrations** page. Then, click on the plus (+) button to add a new integration. Once the integration is loaded, with your TV turned on and connected to your local network, enter the IP address of your TV and a name of your choice. If your TV needs to be paired, you will be prompted to type the PIN code that will be displayed on it.
 
-- TC-P60S60
-- TC-P65VT30
-- TX-32AS520E
-- TX-32DSX609
-- TX-49DX650B
-- TX-50DX700B
-- TX-55CX700E
-- TX-55CX680B
-- TX-55EXW584
-- TX-65EXW784
-- TX-L42ET50
-- TX-P42STW50
-- TX-P50GT30Y
-- TX-P50GT60E
-- TH-32ES500
-- TX-42AS650
-- TX55ASW654
-
-If your model is not on the list then give it a test, if everything works correctly then add it to the list on [GitHub](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/panasonic_viera.markdown).
-
-Some Panasonic Viera TVs allow Home Assistant to turn them on, if you specify the MAC address with `mac:`.
-
-Note that your TV has to reside in the same network as your Home Assistant instance for this platform to work. If you have multiple network interfaces on your Home Assistant instance, you may need to specify the `broadcast_address`.
-
-To add a TV to your installation, add the following to your `configuration.yaml` file:
+## Manual configuration
+If you prefer to use YAML to set up your Panasonic Viera TV, you can still do it. It also allows for some extra settings.
 
 ```yaml
 # Example configuration.yaml entry
-media_player:
-  - platform: panasonic_viera
-    host: 192.168.0.10
+panasonic_viera:
+  host: 192.168.1.10
```

Comment:
Missed one 😓

```suggestion
  host: YOUR_TV_IP
```

We use placeholders for these. Sorry about that.

Reply:
should I add some description to explain what YOUR_TV_IP should be or that's self-explanatory enough?

Reply:
Precisely because it is a placeholder, it should already be clear that you have to adjust this 😉